modules/openapi-generator/src/main/resources/scala-akka-client/README.mustache

```diff
@@ -65,13 +65,70 @@ libraryDependencies += "{{{groupId}}}" % "{{{artifactId}}}" % "{{{artifactVersio
 
 ## Getting Started
 
+Please follow the [installation](#installation) instruction and execute the following Java code:
+
+```scala
+{{#apiInfo}}{{#apis}}{{#-first}}{{#operations}}{{#operation}}{{#-first}}
+import {{invokerPackage}}._
+import {{modelPackage}}._
+import {{{package}}}.{{{classname}}}
+
+import akka.actor.ActorSystem
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+object {{{classname}}}Example extends App {
+
+    implicit val system: ActorSystem = ActorSystem()
+    import system.dispatcher{{#hasAuthMethods}}
+    {{#authMethods}}{{#isBasic}}{{#isBasicBasic}}
+    // Configure HTTP basic authorization: {{{name}}}
+    implicit val {{{name}}}: BasicCredentials = BasicCredentials("YOUR USERNAME", "YOUR PASSWORD"){{/isBasicBasic}}{{#isBasicBearer}}
+    // Configure HTTP bearer authorization: {{{name}}}
+    implicit val {{{name}}}: BearerToken = BearerToken("BEARER TOKEN"){{/isBasicBearer}}{{/isBasic}}{{#isApiKey}}
+    // Configure API key authorization: {{{name}}}
+    implicit val {{{name}}}: ApiKeyValue = ApiKeyValue("YOUR API KEY"){{/isApiKey}}
+    {{/authMethods}}
+    {{/hasAuthMethods}}
+
+    // Create invoker to execute requests
+    val apiInvoker = ApiInvoker()
+    val apiInstance = {{{classname}}}("{{{basePath}}}"){{#allParams}}
+    val {{{paramName}}}: {{{dataType}}} = {{{example}}} // {{{dataType}}} | {{{description}}}
+    {{/allParams}}
+
+    val request = apiInstance.{{{operationId}}}({{#allParams}}{{{paramName}}}{{^-last}}, {{/-last}}{{/allParams}})
+    val response = apiInvoker.execute(request)
+
+    response.onComplete {
+        case Success(org.openapitools.client.core.ApiResponse(code, content, headers)) =>
```

Comment:
Ideally a normal example could use the import instead of the fully qualified name, but the Petstore Spec also define ApiResponse as one of its models

Reply:
Right. Another approach (used in some other client generators) is to whitelist "ApiResponse" so that the model "ApiResponse" will become "ModelApiResponse" to avoid name collision.